pkg/namespacestore/namespacestore.go

```diff
@@ -321,6 +322,12 @@ func createCommon(cmd *cobra.Command, args []string, storeType nbv1.NSType, popu
 	if len(args) != 1 || args[0] == "" {
 		log.Fatalf(`❌ Missing expected arguments: <namespace-store-name> %s`, cmd.UsageString())
 	}
+	endpoint := util.GetFlagStringOrPrompt(cmd, "endpoint")
+	sigVer, _ := cmd.Flags().GetString("signature-version")
+	u, _ := url.Parse(endpoint)
+	if u.Scheme == "http" && sigVer == "v4" {
+		log.Fatalf("Non-secure endpoint works only with v2 signature-version. Please select signature version v2 for namespacestore")
```

Comment:
shouldn't we also handle this for backingstore as well?

Reply:
@jackyalbo we did the test for backingstore and couldnt reproduce the issue in backingstore, looks like it's an issue only with NSS.

Reply:
this sounds weird, I think we are using the same connection for both namespace and backingstore. Do you know of what flow does it happen in namespacestore?

Reply:
ok, I think the issue is with uploadPart which used only by namespace_s3. So we dont need to add this to backingstore. So resolving this conversation. Thanks